pytorch/training/docker/1.8/py3/Dockerfile.cpu

```diff
@@ -24,7 +24,7 @@ ENV PATH=/opt/conda/bin:$PATH
 ENV SAGEMAKER_TRAINING_MODULE=sagemaker_pytorch_container.training:main
 ENV DGLBACKEND=pytorch
 
-ARG PT_TRAINING_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.8.0_aws/20210306-000727/b5cc095ca218fb9eab43948dc83c7a020ab1db26/cpu/torch-1.8.0-cp36-cp36m-manylinux1_x86_64.whl
+ARG PT_TRAINING_URL=https://aws-pytorch-binaries.s3-us-west-2.amazonaws.com/r1.8.1_aws/20210325-012734/e1343088f0beb99438343e1e99e8d71ffb972b47/cpu/torch-1.8.1-cp36-cp36m-manylinux1_x86_64.whl
 ARG PT_TORCHVISION_URL=https://torchvision-build.s3-us-west-2.amazonaws.com/1.8.0/cpu/torchvision-0.9.0-cp36-cp36m-linux_x86_64.whl
```

Comment:
Could you please confirm that there is no corresponding change required on the Torchvision wheel?

Reply:
@yl-to I think the torchvision binary needs to be rebuilt for the new pytorch binary.

Reply:
there is new version torchvision, working on build new ones @saimidu @tejaschumbalkar